packages/tools/devtools/devtools-view/tsconfig.json

```diff
@@ -1,13 +1,11 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../common/build/build-common/tsconfig.node16.json",
+	"include": ["src/**/*"],
+	"exclude": ["src/test/**/*"],
 	"compilerOptions": {
-		"module": "esnext",
-		"lib": ["ESNext", "DOM"],
 		"rootDir": "src",
-		"outDir": "dist",
-		"composite": true,
-		"jsx": "react",
+		"outDir": "- - invalid - -",
```

Comment:
Probably worth a comment here (maybe inside the `outDir` string) to say that this is just a base file for the esm/cjs tsconfig files?